Src/libCZI_UnitTests/test_metadatabuilder.cpp

```diff
@@ -951,3 +951,31 @@ TEST(MetadataBuilder, WriteDisplaySettingsForTwoChannelsWithDifferentTintingMode
     EXPECT_TRUE(node->TryGetValue(&color_mode_element));
     EXPECT_STREQ(L"None", color_mode_element.c_str()) << "Incorrect result";
 }
+
+TEST(MetadataBuilder, WriteDisplaySettingsWithTintingModeNoneAndPixelTypeAndCheckResult)
+{
+    // set tinting mode to none, provide a channel-to-pixeltype-map, and expect to find a mode "<PixelType>Bgr24</PixelType>" in the xml
+    const auto metadata_builder = libCZI::CreateMetadataBuilder();
+
+    DisplaySettingsPOD display_settings;
+    ChannelDisplaySettingsPOD channel_display_settings;
+    channel_display_settings.Clear();
+    channel_display_settings.isEnabled = true;
+    channel_display_settings.tintingMode = IDisplaySettings::TintingMode::None;
+    channel_display_settings.tintingColor = Rgb8Color{ 0xff,0,0 };
+    channel_display_settings.blackPoint = 0.3f;
+    channel_display_settings.whitePoint = 0.8f;
+    display_settings.channelDisplaySettings[0] = channel_display_settings;  // set the channel-display-settings for channel 0
+
+    const auto channel_pixel_type_map = map<int, PixelType>{ {0, PixelType::Bgr24} };
+
+    MetadataUtils::WriteDisplaySettings(metadata_builder.get(), DisplaySettingsPOD::CreateIDisplaySettingSp(display_settings).get(), 1, &channel_pixel_type_map);
```

Comment:
Just wondering about a conceptual thing. The WriteDisplaySettings function accepts an IDisplaySettings object that should hold all display settings. However, we are now adding another parameter for mapping the pixel types to the channels. Is there a reason to not include this in the ChannelDisplaySettingsPOD inside the IDisplaySettings object?

Reply:
This was our initial idea, but this isnt really a display setting, as its part of the image pixel information. if i understand correctly, how it should actually work is that, for each channel, the viewer should read the first subblock and get the pixel information from there (as done in zen), but arivis doesnt, and expects a display setting metadata to have this info. 
The reasoning to not add this to IDisplaySettings is that its only a workaround for arivis and shouldnt be part of an conventional display setting

Reply:
Yes, there is a reason, see above -> https://github.com/ZEISS/libczi/pull/58#issuecomment-1604087665 
In a nutshell - imho this node "should be there", and conceptually, imho it should not be part of the "display-settings". So, I did not want to have the pixel-type added to the display-settings (and that's why it is "injected" here manually).

Reply:
...and yes and for sure - this is rather a "minimal change towards a limited and specific feature" than "a revision of the document-creation and metadata-generation".

Reply:
Ah, that makes sense! Thanks for elaborating!